### ISSUE TITLE
Administrative data is reset on each update from agent

### ISSUE THREAD

**Issue by sreerajp** [OCSInventory-NG/OCSInventory-Server#436]
### General informations
Operating system : Debian 11.5

### Server informations
Perl version :
Mysql / Mariadb / Percona version :  mariadb 15.1

### OCS Inventory informations
Ocs server version : 2.11.1

### Problem's description
Every time the agent sends data to server the Administrative data is reset. Please help

**Comment by sreerajp** [comment]
![image](https://github.com/OCSInventory-NG/OCSInventory-Server/assets/26023313/e46c7fcd-72d5-4989-bb0a-c2b9eb8c293c)

I am getting these in my agent logs. For some agent all 4 are blank. For some only two is blank.

**Comment by sreerajp** [comment]
Can anybody help on this?

Actually we have two OCS Inventory server. One acting as slave and another as master. The issue is in the master. 

We have deployed the slave server with Debian 11.5 OS as VM. Then we took the clone of the slave, cleared its DB except master data and using it as master. Slave is working perfectly fine with Administrative data. I compared both the server and couldn't find any changes. Does anybody have any idea why this is happening? It what context this behavior will happen.

Thank You
